src/EnergyPlus/main.cc

```diff
@@ -50,21 +50,24 @@
 #include <EnergyPlus/Data/EnergyPlusData.hh>
 #include <EnergyPlus/DataStringGlobals.hh>
 #include <EnergyPlus/api/EnergyPlusPgm.hh>
+#include <EnergyPlus/api/state.h>
 
 int main(int argc, const char *argv[])
 {
-    EnergyPlus::EnergyPlusData state;
+    EnergyPlus::EnergyPlusData *state = reinterpret_cast<EnergyPlus::EnergyPlusData*>(stateNew());
```

Comment:
This is indeed interesting.  The elimination of global variable usage certainly cleaned up the possibility of having multiple copies all over the place, but I didn't imagine state itself would have been the culprit in this.  I believe you, but will need to do some testing to also convince myself.  Since you are calling the state API, you can also just use the `EnergyPlusState` typedef to round out the whole situation.